backend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:slim as run
+
+USER root
+
+WORKDIR /usr/src/backend
+COPY package*.json ./
+
+RUN npm install
+
+RUN useradd -ms /bin/bash serviceUser
+
+USER serviceUser
+
+# Change user here otherwise chmod will crash
+CMD "bash" "-c" "cd /usr/src/backend/ && npm run start" 
```

Comment:
npm run start will call nest js....in production we should not do that. we can just execute the already compiled main.js file